### PR TITLE
Updates to the how you compare front page

### PR DESCRIPTION
**Pull request by wasade**
Resolves items from the attached mockup with the exception of:

- Item "3", inserting a switch for 16S <-> shotgun into the breadcrumb bar would require either a modification to `sitebase.html` or some fancy Javascript to resolve. Deferring that item
- A pill for for "microbes and health" was not added as that is pending some additional conversations

how_you_compare

### CODE REVIEW

File: microsetta_interface/templates/new_results_page.jinja2 [hunk 1]
```diff
@@ -1077,7 +1086,17 @@
...
             </div>
           </div>
-
+          <div class="tab-pane fade" id="microbial-map" role="tabpanel" aria-labelledby="microbial-map-tab">
+            <div class="row">
+              <div class="column results_text">
+                <h3>Microbial maps</h3>
+                <br/>
+                <p>
+                foobar
```
Comment (dhakim87) [inline microsetta_interface/templates/new_results_page.jinja2:1095]: This seems less than ideal

**Comment by wasade** [comment]
Microbial maps formatting with draft text

Screen Shot 2021-03-22 at 7 51 55 PM

**Comment by wasade** [comment]
@dhakim87, just fyi, formatting and draft text for microbial maps in with at the moment some placeholder images, some "pcoa-..." ID'd divs.

File: microsetta_interface/templates/new_results_page.jinja2 [hunk 2]
```diff
@@ -74,6 +74,13 @@
     .bs-tooltip-auto[x-placement^=right] .arrow::before, .bs-tooltip-right .arrow::before {
         border-right-color: #fc8900 !important;
     }
+
+    .virtualPlaceholder {
+        display:block;
+        height:500px;
+        width:500px;
+        background:#eee;
+    }
```
Comment (dhakim87) [inline microsetta_interface/templates/new_results_page.jinja2:83]: May have to mess with the sizing, the plotly plots tend to be wider than tall and do weird text cutoffs sometimes, but this is the right ballpark.

Comment (wasade) [reply microsetta_interface/templates/new_results_page.jinja2:83]: Ya figured we'd have to poke it a bunch to get it to look right... but wanted to ahve some place holders in to eyeball how things look

**Comment by wasade** [comment]
fixes #60

**Comment by wasade** [comment]
Merging this as the changes are getting large and its oriented at an example still

**Pull request merged**